l1-contracts/test/Rollup.t.sol

```diff
@@ -78,6 +79,178 @@ contract RollupTest is DecoderBase {
     _;
   }
 
+  function warpToL2Slot(uint256 _slot) public {
+    uint256 initialTime = rollup.GENESIS_TIME();
+    vm.warp(initialTime + _slot * Constants.AZTEC_SLOT_DURATION);
+  }
+
+  function testClaimEpochProofRight() public setUpFor("mixed_block_1") {
```

Comment:
Long test, but it actually seemed like the sanest way to hit the paths.

Reply:
I would prefer different tests for hitting different cases. I know it is not fully what we have been doing in other test, but it makes it simpler to read, and something that was intended for #8324.